src/components/data/DataNavigation.js

```diff
@@ -91,11 +92,7 @@ function pathToRoute(storageId, root, relativePath, selectedPathItemIndex) {
         return accumulator;
     };
     const routerPath = relativePathItems.reduce(reducer);
-    return `${constants.PATH_SEPARATOR}${NavigationConstants.DATA}${
-        constants.PATH_SEPARATOR
-    }${storageId}${root}${constants.PATH_SEPARATOR}${getEncodedPath(
-        routerPath
-    )}`;
+    return `${root}${constants.PATH_SEPARATOR}${routerPath}`;
```

Comment:
I think this still works because now all routing/URL updates go through the `handlePathChange` method in the data pages which will always encode the path.  So if I also encode it here and test with a path with `&` in it, I end up with a double encoding: `http://localhost:3000/data/ds/iplant/home/aramsey/te%2526st`